I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 12, in script
    define config.adv_nvl_transition = NoneS
  File "game/script.rpy", line 12, in <module>
    define config.adv_nvl_transition = NoneS
NameError: name 'NoneS' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 12, in script
    define config.adv_nvl_transition = NoneS
  File "C:\Users\barba\OneDrive\Escritorio\renpy-8.3.2\renpy\ast.py", line 2235, in execute
    self.set()
  File "C:\Users\barba\OneDrive\Escritorio\renpy-8.3.2\renpy\ast.py", line 2249, in set
    value = renpy.python.py_eval_bytecode(self.code.bytecode)
  File "C:\Users\barba\OneDrive\Escritorio\renpy-8.3.2\renpy\python.py", line 1202, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 12, in <module>
    define config.adv_nvl_transition = NoneS
NameError: name 'NoneS' is not defined

Windows-10-10.0.22631 AMD64
Ren'Py 8.3.2.24090902
a_golden_chance 1.0
Sun Nov 10 20:57:02 2024
